yapio.c

```diff
@@ -725,6 +732,33 @@ yapio_parse_test_recipe(const char *recipe_str)
             i += yapio_test_recipe_param_to_ull(&recipe_str[i + 1], &tmp);
             ytg->ytg_num_ranks = (int)tmp;
             break;
+        case 'P':
+            if (recipe_str[i + 1] != '.')
+            {
+                printf("Percentage should start by '.'\n");
+                yapio_print_help(YAPIO_EXIT_ERR);
+            }
+            /* find where percentage stops */
+            int end = i + 2;
```

Comment:
you should use a variable that to store that +2 as it is used in many places, if it chances there is a hight chance of forgetting to update every one. Furthermore it is not very meaning full, and can help to understand your operations if you store that into a variable with a proper name

Reply:
len("P.") = 2, i'll make it clearer in the next version